backend/WebApi/Constants/RouteConstants.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace WebApi.Constants
+{
+    public class RouteConstants
```

Comment:
Strange solution for me. I don't understand why do you need routes as constants...